src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.cpp

```diff
@@ -525,11 +528,13 @@ void SinglePhaseBase::computeHydrostaticEquilibrium()
 
     real64 const eps = 0.1 * (maxElevation - minElevation); // we add a small buffer to only log in the pathological cases
     GEOS_LOG_RANK_0_IF( ( (datumElevation > globalMaxElevation[equilIndex]+eps)  || (datumElevation < globalMinElevation[equilIndex]-eps) ),
-                        SinglePhaseBase::catalogName() << " " << getName()
-                                                       << ": By looking at the elevation of the cell centers in this model, GEOSX found that "
-                                                       << "the min elevation is " << globalMinElevation[equilIndex] << " and the max elevation is " << globalMaxElevation[equilIndex] << "\n"
-                                                       << "But, a datum elevation of " << datumElevation << " was specified in the input file to equilibrate the model.\n "
-                                                       << "The simulation is going to proceed with this out-of-bound datum elevation, but the initial condition may be inaccurate." );
+                        SinglePhaseBase::catalogName() << " " << getDataContext() <<
```

Comment:
sorry for being late with this suggestion
"..Base" classes often do not meaningful catalogName defined leading to some confusing info printed:

`SolverBase compflow (co2_flux_3d.xml, l.5): currently, GEOS assumes that there is only one mobile phase when computing the hydrostatic pressure`

SolverBase here is confusing

Reply:
Would the `Solver` term be clearer or should I totally remove the class name from here ?

Reply:
My point is more general than this specific place: you probably should not use catalogName from "...Base" classes because it is not well defined.

Reply:
Do you think it would be satisfying to use the same mecanism we have in `ConstitutiveBase` ( the `virtual string getCatalogName()` method that gives the instanciated catalogName?

Reply:
Personally, for me printing solver name (getName, seems already included in getDataContext) is good enough to be able to locate the issue in the input file.
What you proposing is also fine, as long as it prints final catalog name and not some intermediate misleading info.
Thanks!

Reply:
https://github.com/GEOS-DEV/GEOS/commit/d184d82682f3d6ebca7dbdfbef2e1d9510a1962f#r130545002 example when getName is simple and straightforward

Reply:
PR is here: #2783  feel free to review :)